scripts/distribution/macOS-package/NodeConfigurationInstallerPlugin/NodeConfigurationInstallerPlugin/MyInstallerPane.m

```diff
@@ -40,7 +40,7 @@ - (BOOL)shouldExitPane:(InstallerSectionDirection)aDir
                 // Initialize the dialog.
                 [warning addButtonWithTitle:@"OK"];
                 [warning setMessageText:@"Invalid node names"];
-                [warning setInformativeText:@"Node names cannot be empty or contain '\"'."];
+                [warning setInformativeText:@"Node names must be between 1 and 100 characters in length and can only contain a-z, A-Z, 0-9, '-', or '_'."];
```

Comment:
```suggestion
                [warning setInformativeText:@"Node names must be between 1 and 100 characters in length and must be a sequence of the following characters: a-z, A-Z, 0-9, '-', or '_'."];
```

Reply:
That was also how I first wrote it, but Lotte suggested the `can only contain` version ;)